framework/source/class/qx/ui/basic/Atom.js

```diff
@@ -293,6 +302,16 @@ qx.Class.define("qx.ui.basic.Atom",
 
 
     // property apply
+    _applySelectable : function(value, old)
+    {
+      var label = this.getChildControl("label", true);
+      if (label) {
+        label.setSelectable(value);
+      }
+    },
+
+
+    // property apply
```

Comment:
Instead of _applySelectable you can bind the value to the label when creating it in _createChildControlImpl. Would save some lines ;-)